Makefile

```diff
@@ -357,11 +357,11 @@ else
 	@rm -rf apache-couchdb-*
 	@$(MAKE) dist 2>&1 > /dev/null
 	@rm apache-couchdb-*.tar.gz
-	@mv apache-couchdb-* apache-couchdb-2.1.0-RC$(COUCH_RC)
-	@tar czf apache-couchdb-2.1.0-RC$(COUCH_RC).tar.gz apache-couchdb-2.1.0-RC$(COUCH_RC)
-	@echo "Done apache-couchdb-2.1.0-RC$(COUCH_RC).tar.gz"
+	@mv apache-couchdb-* apache-couchdb-2.1.1-RC$(COUCH_RC)
```

Comment:
best to use `COUCHDB_VERSION` defined in https://github.com/apache/couchdb/pull/907/files#diff-b67911656ef5d18c4ae36cb6741b7965R18